docs/source/getting-started/path-variables.rst

```diff
@@ -33,28 +33,37 @@ Run the following command in your terminal::
 
         echo 'export JAVA_HOME="$(/usr/libexec/java_home)"' >> ~/.zprofile
 
-In order for the changes to take effect you will need to restart your computer. Note that if you will be setting up the
-``PATH`` variable as well you can restart your computer after you're done with all the changes. Upon restarting check that
-the ``JAVA_HOME`` variable is set::
+Setting the PATH variable
+=========================
+Run the following command in your terminal::
 
-        echo $JAVA_HOME
+        echo 'export PATH="$HOME/.daml/bin:$PATH"' >> ~/.zprofile
 
-The result should be the path to the JDK installation, something like this::
+Verifying the changes
+=====================
 
-        /Library/Java/JavaVirtualMachines/jdk_version_number/Contents/Home
+In order for the changes to take effect you will need to restart your computer, or, if you're using
+the Terminal app, you need to quit the Terminal app (Command+Q in the Terminal window) and
```

Comment:
```suggestion
the MacOS Terminal app, you need to quit the Terminal app (Command+Q in the Terminal window) and
```

Reply:
Makes it a bit clearer that this is a system thing rather than something custom like iterm2.